### DISCUSSION TITLE
Migrate to a new server

### DISCUSSION THREAD

**Discussion by mxwmnn** [coder/code-server#6374] [Q&A] [answered]
Hello,

I'm trying to migrate code-server to a new server.

I rsynced with -arzP to the new server the /root dir and then ran the install script. But there are no extensions installed and the settings aren't available and set like on the old server.

Would love to hear how to migrate the right way.

Thanks!

**Comment by code-asher** [comment, marked as answer]
Yup that is pretty much the right way.  You can get away with only 
syncing the code-server data directory rather than the entire home 
directory, but assuming you are running code-server under the root 
user in both the old and new servers then syncing /root should 
have worked.

The data directory is by default in ~/.local/share/code-server, so 
if running as root that should be /root/.local/share/code-server. 
Can you verify that directory synced properly and that code-server 
is running as root?

**Comment by mxwmnn** [comment]
Did it again with a clean install and now works well with rsync -arvzP from old to new machine.

Thanks !
